bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/graphics/GCData.java

```diff
@@ -51,19 +51,30 @@ public final class GCData {
 	public int fillRule = SWT.FILL_EVEN_ODD;
 	public Image image;
 
+	/** @noreference This field is not intended to be referenced by clients. */
```

Comment:
This should IMHO also be implied by the `@noreference` of this class respectively public non-api types should not be an issue.